features/layout/footer/footer.module.scss

```diff
@@ -0,0 +1,87 @@
+@use "@styles/color";
+@use "@styles/space";
+@use "@styles/breakpoint";
+@use "@styles/z-index";
+@use "@styles/misc";
+
+.footerContainer {
+  background-color: color.$gray-50;
+  padding: 0.75rem;
+
+  @media (min-width: breakpoint.$desktop) {
+    padding: 0;
+  }
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+  gap: 1.5rem;
+  padding: 1.5rem;
```

Comment:
You correctly use the global variables for e.g. `breakpoint` and `color`. It would be important to use them for spacing as well